### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by sky-bones** [sky-bones/portfolio#22]
1. Navigation
   - the navigation doesn't seem to work in without the click action, you can select it but it doesn't do anything. on small sizes, this renders the navigation basically useless. 
2. Screen Reader
   -  When I used the screen reader, it told me there was no content in the html. it would tell me the what was in the title tag, but that was all. I'm thinking it's because it's in a post, but how would I tell the browser where to look for the content? 

@thomasjbradley

**Comment by thomasjbradley** [comment]
_Re: the nav_

We can fix that with a little Javascript.

---

_Re: screen reader_

What browser are you using? Chrome? I've got it working in Firefox right now.

**Comment by sky-bones** [comment]
the nav - excellent! 

screen reader - yep, i'm using chrome. Is there a way around it, or is it just chrome being chrome?

**Comment by thomasjbradley** [comment]
Nope, Chrome is stupid and doesn't work well with VoiceOver.
